cluster/up.sh

```diff
@@ -13,6 +13,6 @@ echo 'The cluster is ready!'
 echo 'Use following command to install macvtap-cni on the cluster:'
 echo '  make cluster-sync'
 echo 'Use following command to access cluster API:'
-echo '  ./kubevirtci/cluster-up/kubectl.sh get nodes'
+echo '  ./_kubevirtci/cluster-up/kubectl.sh get nodes'
```

Comment:
This should be actually pointing to https://github.com/kubevirt/macvtap-cni/blob/master/cluster/kubectl.sh

Reply:
Done.